src/unix/kqueue.c

```diff
@@ -230,10 +244,13 @@ void uv__io_poll(uv_loop_t* loop, int timeout) {
 
       revents = 0;
 
-      if (ev->filter == EVFILT_READ) {
+      if (ev->filter == EVFILT_READ || ev->filter == EV_OOBAND) {
         if (w->pevents & POLLIN) {
           revents |= POLLIN;
           w->rcount = ev->data;
+        } else if (w->pevents & UV__POLLPRI) {
+          revents |= UV__POLLPRI;
+          w->rcount = ev->data;
         } else {
```

Comment:
I'm not too sure this logic is correct. If `ev->filter == EV_OOBAND` and `w->pevents = POLLIN | POLLPRI` couldn't it report a non-existent `POLLIN` event?  I would handle the `UV__POLLPRI` in a separate block as it's now with `POLLIN` and `POLLOUT`.